src/generators/app-lit-element-ts/templates/static-demoing/.storybook/server.mjs

```diff
@@ -0,0 +1,8 @@
+import { storybookPlugin } from '@web/dev-server-storybook';
+import baseConfig from '../web-dev-server.config.mjs';
+
+export default /** @type {import('@web/dev-server').DevServerConfig} */ ({
+  ...baseConfig,
+  open: '/',
+  plugins: [...baseConfig.plugins, storybookPlugin({ type: 'web-components' })],
```

Comment:
`...baseConfig` gets us the HRM from the other config, right?

Reply:
Right, that would be bad. Will need to remove this

Reply:
Hehe, I misread it. But for storybook it's correct that it will inherit all plugins including HMR - as intended. 

But I also did the config merging for WTR, where the HMR would be bad. I removed that now, and added some more configuration hints.